pkg/logging/init.go

```diff
@@ -220,7 +222,7 @@ func getFields(ctx context.Context) (loggerFields, bool) {
 // the context-enriched logger.
 //
 // The strength of this approach is that labels set in parent context are accessible
-func WithFields(ctx context.Context, fields ...interface{}) context.Context {
+func WithFields(ctx context.Context, fields ...slog.Attr) context.Context {
```

Comment:
Ah, it's that simple, noice! Thanks for looking into this, my knowledge about `slog` apparently was / is too limited to see the obvious solution here... :see_no_evil: